packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -918,6 +926,8 @@ class DefaultTextStyleTransition extends AnimatedWidget {
 /// ```
 /// {@end-tool}
 ///
+/// {@animation 300 300 https://flutter.github.io/assets-for-api-docs/assets/widgets/aniamted_builder.mp4}
```

Comment:
Uber nit:
```suggestion
/// {@animation 300 300 https://flutter.github.io/assets-for-api-docs/assets/widgets/animated_builder.mp4}
```
It looks like this typo would need to be fixed in the assets repo first. It links to the proper asset so I think it's ok as is?

Reply:
Fixing spelling here: https://github.com/flutter/assets-for-api-docs/pull/109